erpnext_documentation/www/docs/user/manual/en/automation/event_streaming.md

```diff
@@ -119,11 +124,46 @@ In the Field Mapping child table:
 
 ![Document Type Mapping](/docs/assets/img/automation/event-field-mapping.png)
 
-- **Is Child Table**: If the field you are trying to map is a child table, you need to check this.
-- **Child Table Mapping**: Create another Document Type Mapping for the child table fields and select it here.
+#### 3.5.2 Default Value for some field
+
+If your field is not mapped to any other remote fieldname and you always want the field to have the same value, set the default value field. Event if you have set the remote fieldname, in case during the sync, remote field's value is not found and the "Default Value" has been specified, it will be set.
+
+![Child Table Mapping Link](/docs/assets/img/automation/default.png)
+
+
+#### 3.5.3 Mapping for DocTypes having child tables
+
+If the field you are trying to map is a child table, you need create another Document Type Mapping for the child table fields.
+
+![Child Table Mapping Link](/docs/assets/img/automation/child_table_map_doc.png)
+
+- **Mapping Type**: Select the Mapping Type as Child Table.
+- **Mapping**: Select the Document Type Mapping doc you created for the child table.
+
+![Child Table Mapping Link](/docs/assets/img/automation/event-map-is-child-table.png)
+
+#### 3.5.4 Mapping for DocTypes having dependencies (Link, Dynamic Link fields)
+
+If the DocTypes you are trying to map, have any kind of dependencies like Link or Dynamic Link fields you need to set up another Document Type Mapping for syncing the dependencies.
```

Comment:
If the DocTypes you are trying to map, <-(remove comma)
have any kind of dependencies like Link or Dynamic Link fields, <-add comma